tslearn/metrics.py

```diff
@@ -400,7 +433,7 @@ def dtw_subsequence_path(subseq, longseq):
 
     Compared to traditional DTW, here, border constraints on admissible paths
     :math:`P` are relaxed such that :math:`P_0 = (0, ?)` and
-    :math:`P_L = (N-1, ?)` where :math:`L` is the length of the considered path
+    :math:`P_0 = (0, ?)` where :math:`L` is the length of the considered path
```

Comment:
Hum, I don't agree with your edit. I think the original formulation was the correct one.

Reply:
I don't know why I changed that. You are absolutely right